native/src/components/CitySelector.tsx

```diff
@@ -76,7 +83,9 @@ const CitySelector = ({ cities, navigateToDashboard }: CitySelectorProps): React
           <CityGroup>{t('nearbyCities')}</CityGroup>
           <NearbyCities cities={cities} navigateToDashboard={navigateToDashboard} filterText={filterText} />
         </CityGroupContainer>
-        <SearchCounter>{t('search:searchResultsCount', { count: resultCities.length })}</SearchCounter>
+        <SearchCounter accessibilityLiveRegion={resultCities.length === 0 ? 'assertive' : 'polite'}>
```

Comment:
Interesting. I thought I did but I might have been imagining it. According to https://developer.android.com/reference/kotlin/androidx/compose/ui/semantics/LiveRegionMode, the main difference is that an assertive live region should also interrupt ongoing speech and a polite one shouldn't. Maybe it's because there was no ongoing speech at the moment? Nothing reading out the key that you had tapped or something?

Reply:
Hmm, maybe I am also just not used to talkback yet and use it wrong but just checked and its still this way kinda.... Perhaps you can try again yourself?